### ISSUE TITLE
Json output format doesn't print skipped count in summary section

### ISSUE THREAD

**Issue by sandeepsharmadevops** [goss-org/goss#812]
**Describe the bug**

I am using goss/dgoss to test the Dockerfile, using JSON format to print execution detail. But in the JSON output, we don't get the skipped count details in the summary section, rather skipped details are printed in individual test cases sections.

- json summary section:
```text
  "summary": {
    "failed-count": 0,
    "summary-line": "Count: 9, Failed: 0, Duration: 0.003s",
    "test-count": 9,
    "total-duration": 3305800
  }
```
- json individual test cases execution details
```text
    {
      "duration": 17600,
      "err": null,
      "expected": [
        "\"file\""
      ],
      "found": [
        "\"file\""
      ],
      "human": "",
      "meta": null,
      "property": "filetype",
      "resource-id": "/opt/mydir/myfile.yaml",
      "resource-type": "File",
      "result": 0,
      **"skipped": false,**
      "successful": true,
      "summary-line": "File: /opt/mydir/myfile.yaml: filetype: matches expectation: [\"file\"]",
      "test-type": 0,
      "title": ""
    }
```
However, when we use the documentation output format we do see skipped details in output.

```text
Total Duration: 0.004s
Count: 9, Failed: 0, Skipped: 3
```

**How To Reproduce**

Change the output format to json and then to documentation.

**Expected Behavior**

Should be able to get skipped count in the json output summary section likewise getting in other spported formats.

**Actual Behavior**

**Environment:**
 - Version of goss
 - OS/Distribution version (if applicable)

Let me know in case of an otherwise situation, or else guide me in the right direction.

Thanks!!!

**Comment by sandeepsharmadevops** [comment]
created this PR, please review and merge it if it makes sense. Thanks!!

https://github.com/goss-org/goss/pull/813/files

**Comment by aelsabbahy** [comment]
Closed by #813
